lib/geordi/util.rb

```diff
@@ -62,21 +62,29 @@ def system!(*commands)
         end
       end
 
+      def binstub(executable, arguments='')
+        binstub_file = "bin/#{executable}"
+
+        command = 'bundle exec '
+        command << File.exists?(binstub_file) ? binstub_file : executable
+        command << ' ' + arguments
```

Comment:
Elegant solution :+1: 

I forgot to mention: `bundle exec` is not required when using binstubs as well.